Repository/PhpcrOdmRepository.php

```diff
@@ -100,6 +91,27 @@ public function getTags()
     /**
      * {@inheritdoc}
      */
+    public function reorderNode($sourcePath, $position)
+    {
+        $document = $this->getDocument($sourcePath);
+        $node = $this->getManager()->getNodeForDocument($document);
+        $parent = $node->getParent();
+        $nodeNames = $parent->getNodeNames();
+        $parentDocument = $this->getManager()->find(null, $parent->getPath());
+
+        if (isset($nodeNames[$position + 1])) {
+            $this->getManager()->reorder($parentDocument, $node->getName(), $nodeNames[$position + 1], true);
+        } else {
```

Comment:
yes - It is in the phpdoc of the `EditableRepository#reorder` interface.

On Mon, Jun 27, 2016 at 01:00:05AM -0700, David Buchmann wrote:
>    In [1]Repository/PhpcrOdmRepository.php:
> 
>  > @@ -100,6 +91,27 @@ public function getTags()
>  >      /**
>  >       * {@inheritdoc}
>  >       */
>  > +    public function reorderNode($sourcePath, $position)
>  > +    {
>  > +        $document = $this->getDocument($sourcePath);
>  > +        $node = $this->getManager()->getNodeForDocument($document);
>  > +        $parent = $node->getParent();
>  > +        $nodeNames = $parent->getNodeNames();
>  > +        $parentDocument = $this->getManager()->find(null, $parent->getPath());
>  > +
>  > +        if (isset($nodeNames[$position + 1])) {
>  > +            $this->getManager()->reorder($parentDocument, $node->getName(), $nodeNames[$position + 1], true);
>  > +        } else {
> 
>    does that mean that any position beyond the end just puts the node at the
>    end and not raise an error? i think this makes sense, but needs to be
>    documented.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly, [2]view it on GitHub, or [3]mute the thread.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource/pull/28#discussion_r68535670
>    2. https://github.com/symfony-cmf/resource/pull/28/files/dec12dddccee50a04d6834eff3ffbb06d016dc52#r68535670
>    3. https://github.com/notifications/unsubscribe/AAgZcdKP7sZ0eyNiV7UNp7VZNHTYAmWRks5qP4MFgaJpZM4I9hWA
>    4. https://github.com/symfony-cmf/resource/pull/28/files/dec12dddccee50a04d6834eff3ffbb06d016dc52#r68535670